src/analysis/locally_convex/bounded.lean

```diff
@@ -0,0 +1,117 @@
+/-
+Copyright (c) 2022 Moritz Doll. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Moritz Doll
+-/
+import analysis.normed.normed_field
+import analysis.locally_convex.basic
+import topology.algebra.module.basic
+import topology.bornology.basic
+
+/-!
+# Natural Boundedness
+
+This file defines natural or von Neumann bounded sets and proves elementary properties.
+
+## Main declarations
+
+* `is_bounded`: A set `s` is bounded if every neighborhood of zero absorbs `s`.
+* `natural_bornology`: The set of bounded sets forms a bornology.
```

Comment:
```suggestion
* `is_vonN_bounded`: A set `s` is von Neumann-bounded if every neighborhood of zero absorbs `s`.
* `vonN_bornology`: The bornology made of the von Neumann-bounded sets.
```

Reply:
Please don't forget to also make these be fully qualified names, like `bornology.is_vonN_bounded` if I'm not mistaken. This is necessary for the generated documentation to turn them into links.